core/store/benches/store_bench.rs

```diff
@@ -0,0 +1,83 @@
+#[macro_use]
+extern crate bencher;
+
+use bencher::{black_box, Bencher};
+use near_primitives::borsh::maybestd::sync::Arc;
+use near_primitives::errors::StorageError;
+use near_store::db::DBCol::ColBlockMerkleTree;
+use near_store::{create_store, Store};
+use std::time::{Duration, Instant};
+
+// try to write to db `10m` keys and then read all of them in random order
+fn benchmark_write_then_read_successful(bench: &mut Bencher) {
```

Comment:
it might be helpful to make the number of keys an argument

Reply:
That's a great idea. I added more keys as well.